server/checks/Approval.js

```diff
@@ -323,7 +352,19 @@ export default class Approval extends Check {
     const user = repository.owner.login
     const upstreamComments = await this.github.getComments(user, repository.name, pull_request.number, formatDate(last_push), token)
     const comments = [...frozenComments, ...upstreamComments].filter((c, idx, array) => idx === array.findIndex(c2 => c.id === c2.id))
-    return await this.countApprovalsAndVetos(repository, pull_request, comments, config.approvals, token)
+
+    const upstreamReviews = await this.github.getReviews(user, repository.name, pull_request.number, formatDate(last_push), token)
+
+    const entries = [...comments, ...upstreamReviews].sort((a, b) => {
```

Comment:
Comments and reviews are being combined and sorted to check if any of the -1 are overwritten by the later native `Approval`.